src/ai/backend/install/context.py

```diff
@@ -838,11 +838,11 @@ def hydrate_install_info(self) -> InstallInfo:
             agent_watcher_addr=ServerAddr(HostPortPair("127.0.0.1", 6019)),
             agent_ipc_base_path="ipc/agent",
             agent_var_base_path="var/agent",
-            storage_proxy_manager_facing_addr=ServerAddr(HostPortPair("127.0.0.1", 6021)),
             storage_proxy_client_facing_addr=ServerAddr(
-                bind=HostPortPair(public_component_bind_address, 6022),
-                face=HostPortPair(public_facing_address, 6022),
+                bind=HostPortPair(public_component_bind_address, 6021),
+                face=HostPortPair(public_facing_address, 6021),
             ),
+            storage_proxy_manager_facing_addr=ServerAddr(HostPortPair("127.0.0.1", 6022)),
```

Comment:
Can you please check if the code currently using ports 6021 and 6022 in backend.ai is okay? @Yaminyam

Reply:
This part is irrelevant to the behavior because it is the part that generates the config file.
As mentioned in the issue, we change it to be consistent with the port we use by default.